<?xml version="1.0"?>

<launch>
<arg name="test_mode" default="False"/>
  
  <node pkg="pwm_motor_driver" type="motor_driver.py" name="motor_driver" respawn="true" output="screen">
    <param name="test_mode" value="$(arg test_mode)" />
    <param name="param_path" value="$(find pwm_motor_driver)/params/motor_params.yaml"/>
  </node>
</launch>
